app/lib/features/activities/widgets/session_card.dart

```diff
@@ -0,0 +1,156 @@
+    final lastSeenTs = deviceRecord.lastSeenTs();
+    if (lastSeenTs != null) {
+      final dateTime = DateTime.fromMillisecondsSinceEpoch(
+        lastSeenTs,
+        isUtc: true,
+      );
+      fields.add(dateTime.toString());
+    }
+    final lastSeenIp = deviceRecord.lastSeenIp();
+    if (lastSeenIp != null) {
+      fields.add(lastSeenIp);
+    }
+    fields.add(deviceRecord.deviceId().toString());
+    return Card(
+      margin: const EdgeInsets.symmetric(vertical: 1),
+      child: ListTile(
+        leading: isVerified
+            ? Icon(
+                Icons.verified_rounded,
+                color: Theme.of(context).colorScheme.success,
+              )
+            : Icon(
+                Icons.question_mark_rounded,
+                color: Theme.of(context).colorScheme.error,
+              ),
+        title: Text(deviceRecord.displayName() ?? ''),
+        subtitle: Text(fields.join(' - ')),
+        trailing: PopupMenuButton(
+          itemBuilder: (BuildContext ctx) => <PopupMenuEntry>[
+            PopupMenuItem(
+              onTap: () async => await onLogout(ctx, ref),
+              child: Row(
+                children: [
+                  const Icon(Atlas.exit_thin),
+                  Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 5),
+                    child: Text(
+                      AppLocalizations.of(ctx)!.logOut,
+                      style: Theme.of(ctx).textTheme.labelSmall,
+                      softWrap: false,
+                    ),
+                  ),
+                ],
+              ),
+            ),
+            PopupMenuItem(
+              onTap: () async => await onVerify(ctx, ref),
+              child: Row(
+                children: [
+                  const Icon(Atlas.shield_exclamation_thin),
+                  Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 5),
+                    child: Text(
+                      AppLocalizations.of(ctx)!.verifySession,
+                      style: Theme.of(ctx).textTheme.labelSmall,
+                      softWrap: false,
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Future<void> onLogout(BuildContext context, WidgetRef ref) async {
+    TextEditingController usernameController = TextEditingController();
+    TextEditingController passwordController = TextEditingController();
+    final result = await showDialog<bool>(
+      context: context,
+      builder: (BuildContext ctx) {
+        return AlertDialog(
+          title: const Text('Auth data needed'),
+          content: Wrap(
+            children: [
+              const Text('Please input username.'),
+              TextField(
+                controller: usernameController,
+                decoration: const InputDecoration(hintText: 'Username'),
+              ),
```

Comment:
needing the username seems pointless here ... we can just fetch that from the client ... asking for the password: :+1: !

Reply:
Fixed by https://github.com/acterglobal/a3/pull/852/commits/66b5eb67c43e4656b698beb7d8f719bd8c649f97